src/main/java/edu/byu/cs/autograder/Grader.java

```diff
@@ -112,6 +89,45 @@ public void run() {
         }
     }
 
+    private Map<String, Rubric.Results> evaluateProject(RubricConfig rubricConfig) throws GradingException {
+        // NOTE: Ideally these would be treated with enum types. That will need to be improved with #300.
```

Comment:
Note that there is still lots of room for improvement as in #300, but this is only a lightweight adjustment for now.